src/corelibs/U2View/src/ov_sequence/GSequenceLineViewAnnotated.cpp

```diff
@@ -245,8 +244,9 @@ void GSequenceLineViewAnnotated::mouseDoubleClickEvent(QMouseEvent *me) {
         ctx->emitClearSelectedAnnotationRegions();
     }
     const QVector<U2Region> annotationRegions = annotation->getRegions();
+    int renderAreaPos = renderArea->coordToPos(renderAreaPoint);
```

Comment:
Please change int to qint64 and I will approve

Reply:
OK